Source/TextToSpeechV1/TextToSpeech.swift

```diff
@@ -266,6 +266,427 @@ public class TextToSpeech {
         
     }
     
+    // MARK: - Customizations
+    
+    /**
+     Lists metadata, such as name and description, for the custom voice models that you own.
+     
+     You can use the language query parameter to list voice models for the specified language, or
+     omit the parameter to see all voice models that you own for all languages.
+     
+     - parameter language: The language of the voice models that you want listed.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with an array of information about your custom voice 
+            models.
+     */
+    public func getCustomizations(
+        language: String? = nil,
```

Comment:
Ahh. I'll reword the comment to make it clearer. Leaving language as nil should just return all custom voice models for all languages.